Clarify reset parameter meaning for dynamic quota algorithms (#150)
ietf-wg-httpapi/ratelimit-headers: draft-ietf-httpapi-ratelimit-headers.md

~~~

> If a response contains both the Retry-After and the RateLimit header fields, the reset parameter value SHOULD reference the same point in time as when additional quota becomes available, which SHOULD align with the Retry-After field value.

Reviewer: ```suggestion
If a response contains both the Retry-After and the RateLimit header fields, the Retry-After field value SHOULD NOT reference a point in time earlier than the reset parameter.
```
